CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 7.4.0
```

Comment:
I did the major bump since I also changed the retry on shutdown behavior to infinite. WDYT?